customTypes.ts

```diff
@@ -0,0 +1,32 @@
+export type AvailableDay = { value: number; label: string; checked: boolean };
+
+export type SettingsFormValues = {
+    businessName: string;
+    availableDays: AvailableDay[];
+    deposit: string;
+    operatingHoursStart: string;
+    operatingHoursEnd: string;
+    clientEmailNotifications: boolean;
+    clientSMSNotifications: boolean;
+};
+
+export type ProfileFormValues = {
+    name: string;
+    email: string;
+    phone: string;
+};
+
+export type FirstRunValues = {
```

Comment:
I think we might be able to use an interface and extend the other types
[This is kinda what I'm thinking](https://stackoverflow.com/questions/41385059/possible-to-extend-types-in-typescript)